### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by tadatuta**
I deployed the version with this changes to testing already.

cc @vithar

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 **/*.deps.js
-**/*.bemjson.js
```
Comment (tadatuta) [inline .eslintignore]: хитрый ход ;)

**Comment by tadatuta** [comment]
related to codestyle fix:
https://github.com/bem-site/gorshochek/pull/55
https://github.com/gulp-bem/node-eval/pull/1

File: .eslintignore (hunk 1 above)
Comment (zxqfox) [inline .eslintignore]: ого

File: content/methodology/methodology.en.bemjson.js [hunk 2]
```diff
@@ -485,31 +485,31 @@
                     elem: 'company',
                     content: 'Yandex',
                     url: 'https://tech.yandex.com/bem/',
-                    src: 'promo-companies__yandex.en.svg',
+                    src: 'promo-companies__yandex.en.svg'
```
Comment (zxqfox) [inline content/methodology/methodology.en.bemjson.js]: fuuu, может быть наоборот сделаем?

Comment (tadatuta) [inline content/methodology/methodology.en.bemjson.js]: не раньше, чем https://github.com/blond/eslint-config-pedant/issues/14

**Comment by tadatuta** [comment]
🆙

**Pull request merged**